src/pretix/static/pretixpresale/scss/_cart.scss

```
@@ -103,7 +109,7 @@
 .cart .firstchild-in-panel .cart-row:first-child {
     padding-top: 0;
 }
-.cart [role=rowgroup]:last-child {
+.cart .rowgroup-total:last-child {
```

I think this an improvement to the previous solution which applied a border to any last rowgroup (of course, it would have needed an update to make sure the last rowgroup is not the first one as well). But what we really want, is a line before the total. I just suggest a slight name change.

```suggestion
.cart .cart-rowgroup-total {
```